deployment/stages/staging.rb

```diff
@@ -13,3 +13,4 @@
 set :keep_releases,  2
 
 before 'deploy:setup', 'sumo:setup:client_folder'
+before 'deploy:setup', 'sumo:db:create'
```

Comment:
Nee, want die zijn specifiek voor onze setup. En enkel staging hebben we zeker onder controle.